templates/candlepin.container

```diff
@@ -0,0 +1,25 @@
+[Unit]
+Description=Candlepin
+After=local-fs.target
+
+[Container]
+Image=quay.io/ehelms/candlepin:4.3.12
+PodmanArgs=--cgroups=enabled
+LogDriver=journald
+Network=host
+
+Volume=/etc/tomcat/logging.properties:/etc/tomcat/logging.properties
+Volume=/etc/tomcat/server.xml:/etc/tomcat/server.xml
+Volume=/etc/tomcat/login.config:/etc/tomcat/login.config
+Volume=/etc/tomcat/cert-roles.properties:/etc/tomcat/cert-roles.properties
+Volume=/etc/tomcat/cert-users.properties:/etc/tomcat/cert-users.properties
+Volume=/etc/tomcat/conf.d/jaas.conf:/etc/tomcat/conf.d/jaas.conf
+Volume=/etc/tomcat/tomcat.conf:/etc/tomcat/tomcat.conf
```

Comment:
My reasoning behind using `tomcat@` on Matrix was that we can use `/etc/candlepin/tomcat` or similar instead of `/etc/tomcat`. Then you can just mount that whole directory.

Perhaps we can start with using `$tomcat_conf` in here instead of hardcoding `/etc/tomcat` to make it portable.

Reply:
> Perhaps we can start with using `$tomcat_conf` in here instead of hardcoding `/etc/tomcat` to make it portable

We can definitely move this to be a template using variables, that was a next step. First step was just getting it to pass tests (which it finally did).



> My reasoning behind using `tomcat@` on Matrix was that we can use `/etc/candlepin/tomcat` or similar instead of `/etc/tomcat`. Then you can just mount that whole directory.

I don't think I understand what the config directly has to do with the service name, but, honestly I would rather just the service be called `candlepin` and abstract away the tomcat part.

Reply:
On EL 8+ you have `tomcat.service` which is a global since instance and `tomcat@.service` which uses `/var/lib/tomcats/$NAME` instead of `/etc/tomcat`. It's the multi-instance solution.

And I agree `candlepin.service` would be better. You can probably create a systemd drop in that adds the alias for you so you can use `systemctl $action candlepin`.